### ISSUE TITLE
dealing with CSS by parsing it into "style"

### ISSUE THREAD

**Issue by foice** [paolo-gurisatti/tk_html_widgets#4]
I am reading into a `HTMLScrolledText` a whole piece of HTML with CSS from file 
```
help_doc="GUI_HELP.html"  #from head to the end
help_doc_filehandle = open(help_doc, "r")
soup = BeautifulSoup(help_doc_filehandle, 'html.parser')
keyword_entry_help=HTMLScrolledText(content, html=soup.decode_contents())
```
This gives unformatted text, but I was thinking of a workaround as It seems that this `HTMLScrolledText` widget can handle `style`.  So I was wondering about any existing tool that might automate the "removal" of CSS in the HTML. 

This would take strings like 
```
<li><span class="code">[seq] </span> matches any character in seq</li>
```
and turn them into "explicit style" string
```
<li><span style=\"background-color: LightGray;\">[seq] </span> matches any character in seq</li>
```

Af course the HTML file we are reading must provide somewhere 
```
.code {
  background-color: LightGray;
}
```
Basically it should be a string replacement 
` class="code"` into  `style=\"background-color: LightGray;\"`
and it might work for a relatively large set of CSS instructions.

Do you have any thoughts on 
- existing tools to make CSS explicit as style (I am attempting to write a simple one, but feels like reinveinting the wheel)
- other ways to support CSS in `HTMLScrolledText`?
